guides/v2.1/architecture/archi_perspectives/framework.md

```diff
@@ -33,13 +33,15 @@ The Framework is responsible for operations that are useful for potentially all
 Here is the Magento Framework folder structure:
 
 
-`vendor/
+```bash
```

Comment:
file tree doesn't relate to Bash
change to `tree` or `terminal`

Reply:
Updated but removed file from this PR.

Reply:
Nothing changed